src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -969,6 +971,24 @@ def _wrapper(func):
             return func
         return _wrapper
 
+    @staticmethod
+    def MethodMap(method=None, resource=False, status=None, version=DEFAULT_VERSION):  # noqa: N802
```

Comment:
Do we need `method=None` as a parameter over here? Because methods like list() or create() are by default GET and POST. I dont think anyone's gonna put 

```
RESTController.MethodMap('GET')
create():
```

Reply:
Done